pkgs/development/interpreters/python/wrapper.nix

```diff
@@ -22,29 +22,39 @@ let
     inherit ignoreCollisions;
     extraOutputsToInstall = [ "out" ] ++ extraOutputsToInstall;
 
-    postBuild = ''
+    buildInputs = [ python.pkgs.propagateWrapperArgsHook ];
+
+    postBuild = with stdenv.lib; ''
       . "${makeWrapper}/nix-support/setup-hook"
 
+      # Load propagated wrapper arguments from each
+      # path in the environment.
+      loadWrapperArgsHook ${concatStringsSep " " paths}
+
       if [ -L "$out/bin" ]; then
           unlink "$out/bin"
       fi
       mkdir -p "$out/bin"
 
-      for path in ${stdenv.lib.concatStringsSep " " paths}; do
+      for path in ${concatStringsSep " " paths}; do
         if [ -d "$path/bin" ]; then
           cd "$path/bin"
           for prg in *; do
             if [ -f "$prg" ]; then
               rm -f "$out/bin/$prg"
               if [ -x "$prg" ]; then
-                makeWrapper "$path/bin/$prg" "$out/bin/$prg" --set NIX_PYTHONPREFIX "$out" --set NIX_PYTHONEXECUTABLE ${pythonExecutable} --set NIX_PYTHONPATH ${pythonPath} ${if permitUserSite then "" else ''--set PYTHONNOUSERSITE "true"''} ${stdenv.lib.concatStringsSep " " makeWrapperArgs}
```

Comment:
ah `--set NIX_PYTHONPREFIX "$out"` got lost

Reply:
Thank you, fixed.